include/deal.II/sundials/arkode.h

```diff
@@ -490,8 +493,11 @@ namespace SUNDIALS
      * @param data ARKode configuration data
      * @param mpi_comm MPI communicator
      */
-    ARKode(const AdditionalData &data=AdditionalData(),
-           const MPI_Comm mpi_comm = MPI_COMM_WORLD);
+    ARKode(const AdditionalData &data=AdditionalData()
+#ifdef DEAL_II_WITH_MPI
+                                      , const MPI_Comm mpi_comm = MPI_COMM_WORLD
```

Comment:
Changing the class interface based on DEAL_II_WITH_MPI is not ideal. I would think we should just ignore that argument if in serial. Can you add a comment in the documentation about the role of the communicator if in serial?